nodes/widgets/ui_switch.html

```diff
@@ -194,6 +206,37 @@
             </a>
         </div>
     </div>
+    <div class="form-row">
+        <label style="vertical-align: top"><i class="fa fa-th-large"></i> <span>Layout</span></label>
+        <div style="display:inline-block">
+            <input type="hidden" id="node-input-layout">
+            <input type="hidden" id="node-input-layoutAlign" >
+            <div>
+                <a href="#" class="nr-db-switch-layout nr-db-switch-layout-row nr-db-switch-layout-row-left">
+                    <span class="nr-db-switch-layout-label">label</span>
+                    <span class="nr-db-switch-layout-value">switch</span>
+                    <div class="nr-db-switch-layout-checkbox"></div>
+                </a>
+                <a href="#" class="nr-db-switch-layout nr-db-switch-layout-row nr-db-switch-layout-row-left-swapped">
+                    <span class="nr-db-switch-layout-value">switch</span>
+                    <span class="nr-db-switch-layout-label">label</span>
+                    <div class="nr-db-switch-layout-checkbox"></div>
+                </a>
+            </div>
+            <div>
+                <a href="#" class="nr-db-switch-layout nr-db-switch-layout-row nr-db-switch-layout-row-spread">
+                    <span class="nr-db-switch-layout-label">label</span>
+                    <span class="nr-db-switch-layout-value">switch</span>
+                    <div class="nr-db-switch-layout-checkbox"></div>
+                </a>
+                <a href="#" class="nr-db-switch-layout nr-db-switch-layout-row nr-db-switch-layout-row-spread-swapped">
+                    <span class="nr-db-switch-layout-value">switch</span>
+                    <span class="nr-db-switch-layout-label">label</span>
+                    <div class="nr-db-switch-layout-checkbox"></div>
+                </a>
+            </div>
```

Comment:
I don't understand _why_, but for some reason, even though this code is correct, it renders in the Node-RED Editor with the last option reversed in it's rendering:

<img width="369" alt="Screenshot 2024-09-24 at 16 46 32" src="https://github.com/user-attachments/assets/35a7b775-21c5-404a-a73a-b169bf0925ba">

So there are two options that _look_ the same. The HTML rendered to the DOM is as rendered, but not as defined in the underlying HTMl file. It's incredibly strange.

Functionally, the PR is great. Docs are good, but this is a strange one

Reply:
I can see the problem, CSS issues. Fixing  myself now, and will use `display: flex` as it's cleaner

Reply:
> I can see the problem, CSS issues. Fixing myself now, and will use `display: flex` as it's cleaner

Thanks fixing this Joe